### ISSUE TITLE
Some Bugs.

### ISSUE THREAD

**Issue by UnknownShifter** [spacebuild/sbep#97]
I play spacebuild a lot, and I'll list a few bugs I have encountered.

1: Two parented objects with docking ports do not dock.  (All of the visuals work, but they do not pull together.)

2: Console commands to change the model of environmental control entities do not work.  (Haven't tested for other entities, but I believe it affects all.)

3: Life Support screens do not display to anyone but the client.

4: Right-Clicking on anything but the node with the Smart Link Tool causes script errors.

Currently that's all I can list off the top of my head.  I'll update the list as I find more bugs.

**Comment by copperwarrior** [comment]
Anything to do with parenting is most likely a direct result of the klunky parenting system, but the other issues listed should be fixable.

**Comment by UnknownShifter** [comment]
Yeah, unfortunately true I suppose.  Parenting is a wonderfully flawed system.

**Comment by copperwarrior** [comment]
New pull request submitted, fixes issue 4.

As for issue 2, the command to change the models are only for the ghosted model, the entities themselves have models associated with them and cannot be changed short of editing the table itself in their current state. 

I am not sure how to handle Issue 3, can one of the other devs take a look at it?
